### PR TITLE
Connect signals

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description 
This branch allows to connect the request signals with the receiver functions defined on the signals.py file.

## Reviewers 
- [x] @felipemontoya 
- [x] @Squirrel18 
- [x] @diegomillan

### CODE REVIEW

File: eox_tenant/apps.py [hunk 1]
```diff
@@ -24,4 +24,22 @@ class EdunextOpenedxExtensionsTenantConfig(AppConfig):
...
             },
         },
+        'signals_config': {
+            'lms.djangoapp': {
+                'relative_path': 'signals',
+                'receivers': [{
+                    'receiver_func_name': 'start_tenant',
+                    'signal_path': 'django.core.signals.request_started',
+                },
+                    {
```
Comment (felipemontoya) [inline eox_tenant/apps.py]: indent

File: eox_tenant/apps.py [hunk 2]
```diff
@@ -24,4 +24,22 @@ class EdunextOpenedxExtensionsTenantConfig(AppConfig):
                 'aws': {'relative_path': 'settings.aws'},
             },
         },
+        'signals_config': {
+            'lms.djangoapp': {
+                'relative_path': 'signals',
+                'receivers': [{
```
Comment (felipemontoya) [inline eox_tenant/apps.py]: In general, this format is a bit off

File: eox_tenant/__init__.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
 """
 Init for campusromero_openedx_extensions.
 """
-__version__ = '0.2.0'
+__version__ = '0.2.1'
```
Comment (felipemontoya) [inline eox_tenant/__init__.py]: This is some serious change.
Candidate for a major bump.

File: eox_tenant/__init__.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
 """
 Init for campusromero_openedx_extensions.
```
Comment (felipemontoya) [inline eox_tenant/__init__.py]: Can you please change this docstring

**Comment by andrey-canon** [comment]
@felipemontoya  I have already applied the requested changes.

**Comment by andrey-canon** [comment]
@felipemontoya If we keep it, the receivers will be executed twice. I think we should merge this, create a release and then create the PR.

**Pull request merged**